CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing Guide
+
+This document describes the requirements for committing to this repository.
+
+## Developer Certificate of Origin (DCO)
+
+In order to contribute to this project, you must sign each of your commits to
+attest that you have the right to contribute that code. This is done with the
+`-s`/`--signoff` flag on `git commit`. More information about DCO can be found
+[here](https://developercertificate.org/)
+
+## Pull Request Management
+
+All code that is contributed to kube-rs must go through the Pull Request (PR)
+process. To contribute a PR, fork this project, create a new branch, make
+changes on that branch, and then use GitHub to open a pull request with your
+changes.
+
+Every PR must be reviewed by at least one Core Maintainer of the project. Once
```

Comment:
Nit: Might be worth defining a list of Core Maintainers somewhere (codeowners file? org members?)

In the latter case, you're still listed as private over at https://github.com/orgs/kube-rs/people.

Reply:
I have updated my membership to public. That was an accident.
Also I have renamed the owners team to `core-maintainers`: https://github.com/orgs/kube-rs/teams/core-maintainers

I'll add a link to it.

Reply:
Doesn't look like the team is public, FWIW.

Reply:
It is set to visible, which is the highest setting. But apparently teams are only visible to org members, so may need to do something else there.